### DISCUSSION TITLE
Photoview does not seem to support image tags, why?

### DISCUSSION THREAD

**Discussion by mkjn72** [photoview/photoview#617] [Ideas]
Hi all,

I've managed to install Photoview in Docker making my first steps towards using it for my local photo collection. I like it a lot, specially the mutliple 'entrypoints' to images like people, albums, places etc. But what seems to be missing is tag support?
I would love to be able to navigate my collection via tags, are there any plans to support this in the near future? Would be great!

**Comment by viktorstrate** [comment]
I'm glad you enjoy it! Would you care to elaborate on what you mean when you say "tags", just to make sure we're on the same page, as that term is a little broad.

Here are some issues that might be related #145 #453 #94

**Reply by mkjn72** [reply]
Thanks for your quick reply. Well, what I was planning to do is add metadata or keywords to my photo files, so I can use multiple 'entrypoints' per file. See attached screenshot of a windows file properties:

![Metadata tag example](https://user-images.githubusercontent.com/68860845/144121273-412b4546-f9fe-490c-8ea1-510c4fcbfee5.png)
 
Excuse the dutch, but as you can see I added tags like:

'Bonnetje' (meaning receipt btw)
'Parkeren'

etc. These are randomly chosen, but it would be fantastic if this metadata was accessible via photoview. So that photoview could generate automagical 'albums' (image galleries) based on these tags. Maybe even with an old school tagcloud or something. But I'm not a UI expert :-)

Regarding the issues you referred to: looks like #145 is definitely related.

**Reply by viktorstrate** [reply]
Yeah, I think that issue is exactly what you're looking for, you can give the issue a 👍, I often sort by that to figure out what to work on next. The next big thing I'm thinking about implementing is a better search system, and I think this issue would be great if it also integrated with search.
